app/controllers/document_associations_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class DocumentAssociationsController < ApplicationController
+  attr_reader :document
```

Comment:
I had to look at this a couple of times to understand what was happening here (`check_document_supports_associations` loads `@document` as a side effect, reader exposes `document` to subsequent actions). Do we need to check if the document has associations -  I think all formats would have to be tagged to organisations? The code might be simplified if we don't need to check.

Reply:
Yeah it's never great for readability using before_action stuff. I could set up a set_document one first if that'd improve readability for you? - I find the before_action stuff all a bit icky

As for associations. We have a bunch of specialist formats that have implicit associations e.g. aaib reports and wouldn't have any others. We wouldn't need to check for now though while we're just dealing with Whitehall doc types.